src/pages/ItemDetail/ItemDetail.js

```diff
@@ -1,8 +1,157 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './ItemDetail.scss';
 
 const ItemDetail = () => {
-  return <div />;
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    setData(ITEM_DATA);
+  }, []);
+
+  const [quantity, setQuantity] = useState(1);
+  const price = data.price;
```

Comment:
price만 따로 선언해서 사용하고 계신데, 구조분해할당을 통해 전체 데이터를 깔끔하게 사용할 수 있습니다.
```suggestion
  const { categories, name, price, ... } = data;
```